wordpress-mobile/WordPress-iOS
# Update prologue map asset

This PR updates the prologue map asset to use the new blue colouring instead of pink. It also switches the PNG assets out for a single PDF.

| Light | Dark |
|---|---|
| ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-16 at 16 03 23](https://user-images.githubusercontent.com/4780/115046268-2fa9ad00-9ecf-11eb-84c1-d006d3c57ba3.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-04-16 at 16 13 23](https://user-images.githubusercontent.com/4780/115046283-333d3400-9ecf-11eb-8604-ef3338f29dd3.png) |
| ![Simulator Screen Shot - iPad Pro (9 7-inch) - 2021-04-16 at 16 10 47](https://user-images.githubusercontent.com/4780/115046297-359f8e00-9ecf-11eb-8d9b-ab7823af42f0.png) | ![Simulator Screen Shot - iPad Pro (9 7-inch) - 2021-04-16 at 16 13 02](https://user-images.githubusercontent.com/4780/115046308-389a7e80-9ecf-11eb-96f2-a508c2432eaa.png) |

**To test**

- Build and run, log out if necessary
- Swipe through the prologue and check the map is now blue and looks as it should

## Regression Notes
1. Potential unintended areas of impact

None, just updating an asset.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:
